tests/win_test.go

```diff
@@ -338,20 +357,83 @@ fmt.Scanln(&input1)
 
 	}
 
-//debug
-var input2 string
-fmt.Scanln(&input2)
-
-
 
 		})
 
 
 	})
-})
 
 
 
+Context("check visable sockets/cores/threads", func() {
+	It("testcase", func() {
+
+})})
 
 
+Context("check hyperv", func() {
+	It("testcase", func() {
+
+		var vmiHyperv tframework.VirtualMachine
+		vmiHyperv.Manifest = address_common + "/hyperv.yml"
+
+		//workaround (maybe some kind of dev env issue)
+		_, _, err := ktests.RunCommandWithNS(TestSystemNamespace, "oc", "project", TestSystemNamespace)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Create VMI using manifest with HyperV features")
+		vmiName := tframework.CreateVMorVMIfromManifest(vmiHyperv.Manifest, TestSystemNamespace)
 
+		By("Getting VMI object")
+		getVMOptions := metav1.GetOptions{}
+		vmi, err := virtClient.VirtualMachineInstance(ktests.NamespaceTestDefault).Get(vmiName, &getVMOptions)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Waiting until VMI start")
+		ktests.WaitForSuccessfulVMIStart(vmi)
+
+		By("Get vmi XML")
+		vmiXml, err := ktests.GetRunningVirtualMachineInstanceDomainXML(
+			virtClient,
+			vmi,
+		)
+
+		By("unmarshal XML")
+		domStat := &api.DomainSpec{}
+		err = xml.Unmarshal([]byte(vmiXml), domStat)
+		Expect(err).ToNot(HaveOccurred())
+			
+		By("Checking HyperV emulation features for KVM")	
+		features := domStat.Features
+               	hyperv := features.Hyperv
```

Comment:
also this may be nil. In this case the test will crash, but I guess that could count as a failure

Reply:
I've changed it